src/tdastro/opsim/ztf_opsim.py

```diff
@@ -96,11 +96,30 @@ def calculate_ztf_zero_points(
         + 4.0
         * (sky * npix * gain + readnoise**2 * nexposure * npix + darkcurrent * npix * exptime * nexposure)
     )
-    zp = 2.5 * np.log10(flux_at_5sigma_limit / gain) + maglim
+    zp = 2.5 * np.log10(flux_at_5sigma_limit) + maglim
 
     return zp
 
 
+def convert_ztf_zp_mag_to_njy(zp_mag):
+    """Convert the zero point in magnitude to zero point in nJy
+
+    Parameters
+    ----------
+    zp_mag: float or ndarray
+        Zero point that converts ADU (or electrons) to magnitude.
+
+    Returns
+    -------
+    zp_nJy: float or ndarry
+        Zero point that converts ADU (or electrons) to nJy.
+    """
+
+    zp_nJy = np.power(10.0, -0.4 * (zp_mag - 31.4))
+
+    return zp_nJy
+
+
```

Comment:
We have `mag2flux` for that
https://github.com/lincc-frameworks/tdastro/blob/ffdb32e4f626dc18ec2cfebf52db418f28ce6f73/src/tdastro/astro_utils/mag_flux.py#L10